app/stac_api/admin.py

```diff
@@ -30,8 +31,21 @@ class CollectionLinkInline(admin.TabularInline):
 
 
 @admin.register(Collection)
-class CollectionAdmin(admin.ModelAdmin):
+class CollectionAdmin(admin.GeoModelAdmin):
+    Collection._meta.get_field('extent_geometry').editable = True  # to show the field
+    modifiable = False  # to prohibit editing extent_geometry
```

Comment:
this should apply to the `extent_geometry` field only I guess? What's the behaviour if you add it on `CollectionAdmin` class level?